app/config/routes.js

```diff
@@ -0,0 +1 @@
+// routes?
```

Comment:
the routes are just currently defined in the app. I figured we could abstract them out later and put them into this file